gandalf.js

```diff
@@ -1,15 +1,16 @@
+const squaredEvenNumbers = numbers.filter(num => num % 2 === 0).map(num => num ** 2);
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.